sql/core/src/main/scala/org/apache/spark/sql/execution/ui/ExecutionPage.scala

```diff
@@ -168,6 +177,45 @@ class ExecutionPage(parent: SQLTab) extends WebUIPage("execution") with Logging
     <br/>
   }
 
+  private def modifiedPandasOnSparkConfigs(
+      modifiedPandasOnSparkConfigs: Map[String, String]): Seq[Node] = {
+    if (modifiedPandasOnSparkConfigs.isEmpty) return Nil
+
+    val modifiedOptions = modifiedPandasOnSparkConfigs.toSeq.map { case (k, v) =>
+      // Remove prefix.
+      val key = k.slice("pandas_on_Spark.".length, k.length)
+      // The codes below is a simple version of Python's repr().
+      // Pandas API on Spark does not support other types in the options yet.
+      val pyValue = parse(v) match {
+        case JNull => "None"
+        case JBool(v) => v.toString.capitalize
+        case JString(s) => s"'$s'"
+        case _ => v
+      }
+      (key, pyValue)
+    }
+
+    val configs = UIUtils.listingTable(
+      propertyHeader,
+      propertyRow,
+      modifiedOptions.sorted,
+      fixedWidth = true
+    )
+
+    <div>
+      <span class="collapse-pandas-on-spark-properties collapse-table"
+            onClick="collapseTable('collapse-pandas-on-spark-properties',
+             'pandas-on-spark-properties')">
+        <span class="collapse-table-arrow arrow-closed"></span>
+        <a>Pandas API on Spark: Properties</a>
```

Comment:
I will create another PR to rename "SQL Properties" to something like "SQL/DataFrame: Properties" later.

Reply:
How about 
```
Pandas API Properties
```
It's odd to have a `:` in the name